scripts/initialize-build-host.sh

```diff
@@ -153,11 +153,44 @@ then
     fi
 
     # --------------------------------------------------------------------------
-    # Populate build host.
+    # Check nested VM health and reset it, if needed.
     # --------------------------------------------------------------------------
 
     login="$(cat $HOME/proxy-target.txt)"
 
+    if [ -z "$login" ]
+    then
+        echo "Nested VM not started, aborting the build"
+        exit 1
+    fi
+
+    if ! $RSH $login true
+    then
+        echo "Failed to SSH to nested VM, probably it's hanging, resetting it"
+        VM_id="$(sudo virsh list | cut -d' ' -f 2 | sed 's/[^0-9]//g;/^$/d')"
+        sudo virsh reset $VM_id
+        attempts=20
+        while true
+        do
+            if $RSH $login true
+            then
+                echo "Nested VM is back up, it seems. Happily continuing!"
+                break
+            fi
+            attempts=$(($attempts - 1))
```

Comment:
For portability, skip double parentheses, use `expr`

```
attempts=`expr $attempts - 1`
```

Reply:
I don't think we run this script on very exotic platforms, but better be more portable than less